### ISSUE TITLE
Why don't you provide an api like "SimpleRest.setPublicationOptions" ?

### ISSUE THREAD

**Issue by wallfacer005-vip** [stubailo/meteor-rest#134]
You provide an api SimpleRest.setMethodOptions for Meteor methods config, and it's very convenience, but why not a SimpleRest.setPublicationOptions for publication config ? 

I think it is better than add param for Meteor.publish .

**Comment by formspoint** [comment]
Are you going to submit a PR for this?

**Comment by wallfacer005-vip** [comment]
@formspoint yes

**Comment by aadamsx** [comment]
I'm trying to "rebuild" this project over here: https://github.com/aadamsx/restfine.  No more coffee script, refactored to ES6, and now trying to convert the project to npm.  Any and all help is appriciated.

**Comment by stubailo** [comment]
@aadamsx hey, thanks - I think it's clear I haven't had any time to maintain this, and I'd be happy to redirect people to your project.

It's also possible to simply attach an `express` server to your Meteor WebApp, would that be better for a generic API? It seems like it might be better to use some existing REST API framework like Restify or others rather than having a new one?

**Comment by aadamsx** [comment]
Yeah, you're probabbly right.  

I found this article on integrating Meteor with Express (http://www.mhurwi.com/meteor-with-express/), but the Accounts and Auth or HTTP is not flushed out.  Maybe I'll just work that angle for a while and see where it leads.  I just need to get a Meteor API working somehow, and the options avaiable right now all seem to have pitfalls.

I opened an feature request on the Meteor repo here (https://github.com/meteor/meteor-feature-requests/issues/141).

**Comment by stubailo** [comment]
Yeah I'd suggest focusing specifically on the auth angle rather than building the API itself!
